README.md

```diff
@@ -2,53 +2,43 @@
 
 Easily manage routing using Cloudflare Workers
 
-
-
 ## Usage
 
-```typescript
-import { startWorker } from '@dbl-works/cloudflare-router'
-
-startWorker({
-  routes: {
-    'example.com': 's3://eu-central-1.assets.example.com',
-  },
-  edgeCacheTtl: 360 // seconds, Edge Cache TTL (Time to Live) specifies how long to cache a resource in the Cloudflare edge network
-})
+```toml
+# wrangler.toml
+name = "my-worker"
+routes = [
+  { pattern = "*example.com/*", zone_id = "5d7a8c0f96213b4e1a57b0c2f9478ec3" },
+]
```

Comment:
```suggestion
routes = [
  {
    pattern = "*example.com/*",
    zone_id = "5d7a8c0f96213b4e1a57b0c2f9478ec3"
  },
]
```

is this valid TOML syntax? If yes, let's please use 1 line per item to make it easier to read and have simpler git diffs.